lib/adyen/rest.rb

```diff
@@ -26,11 +26,7 @@ module REST
     # @return [Adyen::REST::Client] A configured client instance
     # @see .session
     def self.client
-      Adyen::REST::Client.new(
-        Adyen.configuration.environment,
-        Adyen.configuration.api_username,
-        Adyen.configuration.api_password
-      )
+      Adyen::REST::Client.new(Adyen.configuration.environment)
```

Comment:
That's how Adyen configured my account. I guess it's meant to be more secure, since these two account together could withdraw all the money you've on Adyen.